docs/ansible.md

```diff
@@ -15,9 +15,9 @@ virtualenv  --python=$(which python3) $VENVDIR
 source $VENVDIR/bin/activate
 cd $KUBESPRAYDIR
 pip install -U -r requirements-$ANSIBLE_VERSION.txt
-test -f requirements-$ANSIBLE_VERSION.yml && \
```

Comment:
humm, line 17 installs necessary libraries with update option.
I am confused the line 18+.

@cristicalin Could you help me to understand it?

/hold

Reply:
It does not say anything if the file is there or missing and could be like below if it's really needed 
```
if [ test -f "requirements-$ANSIBLE_VERSION.txt" ]; then
    echo "requirements-$ANSIBLE_VERSION.txt exists."
else 
    echo "requirements-$ANSIBLE_VERSION.txt does not exist."
fi
```

Reply:
Yeah, that can be.
I just want to confirm the `yml` file is intentional or not instead of the `txt` file.
#8827 added the section.

Reply:
I might get the point.
The part seems comming from https://github.com/kubernetes-sigs/kubespray/blob/ecd649846a3762555a69bf950dcf0177bd09e15e/tests/scripts/testcases_prepare.sh#L13-L18

and that is necessary for supporting ansible 2.9 and 2.10 as https://github.com/kubernetes-sigs/kubespray/pull/8827
Now Kubespray support 2.11+. So we just need to drop this part to avoid confusion.

Reply:
/hold cancel